components/finder.rst

```diff
@@ -82,7 +82,7 @@ directory to use for the search::
 Search in several locations by chaining calls to
```

Comment:
Except for one change:

```php
// search inside *both* directories
$finder->in(array(__DIR__, '/elsewhere'));
// same as above
$finder->in(__DIR__)->in('/elsewhere');
```

This is to be clear what calling `in()` twice does - it does not replace the previous directory.